### DISCUSSION TITLE
What would it take to build an insert current date plugin?

### DISCUSSION THREAD

**Discussion by Lastofthefirst** [florisboard/florisboard#2705] [Q&A] [answered]
I would love a button on the smart bar for writing the current date. I see that support for extensions was added, and im thinking this might be a trivial extension to write, but i haven't jumped into the documentation yet. Would this be easy to make? Would i have to write much boilerplate or is there something i could start from and just add the function call and an icon for the bar?

**Comment by Lastofthefirst** [comment]
I wanted to check in about this again, is this feasible, or will it be in the future?

**Reply by patrickgold** [reply, marked as answer]
Not really, as FlorisBoard does only support static extensions, and a currentdate plugin is a dynamic one, requiring to pragmatically execute code to figure out the current date.
